No graphs in arm build

[ERROR] [10/26/2020 13:57:54.955] [default-akka.actor.internal-dispatcher-18] [akka://default/user/$a] null
java.lang.NullPointerException
        at sun.awt.FontConfiguration.getVersion(FontConfiguration.java:1264)
        at sun.awt.FontConfiguration.readFontConfigFile(FontConfiguration.java:219)
        at sun.awt.FontConfiguration.init(FontConfiguration.java:107)
        at sun.awt.X11FontManager.createFontConfiguration(X11FontManager.java:774)
        at sun.font.SunFontManager$2.run(SunFontManager.java:431)
        at java.security.AccessController.doPrivileged(Native Method)
        at sun.font.SunFontManager.<init>(SunFontManager.java:376)
        at sun.awt.FcFontManager.<init>(FcFontManager.java:35)
        at sun.awt.X11FontManager.<init>(X11FontManager.java:57)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at java.lang.Class.newInstance(Class.java:442)
        at sun.font.FontManagerFactory$1.run(FontManagerFactory.java:83)
        at java.security.AccessController.doPrivileged(Native Method)
        at sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
        at sun.font.SunFontManager.getInstance(SunFontManager.java:250)
        at sun.font.FontDesignMetrics.getMetrics(FontDesignMetrics.java:264)
        at sun.java2d.SunGraphics2D.getFontMetrics(SunGraphics2D.java:856)
        at org.jfree.chart.text.G2TextMeasurer.getStringWidth(G2TextMeasurer.java:63)
        at org.jfree.chart.text.TextUtils.nextLineBreak(TextUtils.java:227)
        at org.jfree.chart.text.TextUtils.createTextBlock(TextUtils.java:168)
        at org.jfree.chart.title.TextTitle.arrangeRR(TextTitle.java:628)
        at org.jfree.chart.title.TextTitle.arrange(TextTitle.java:496)
        at org.jfree.chart.JFreeChart.drawTitle(JFreeChart.java:1370)
        at org.jfree.chart.JFreeChart.draw(JFreeChart.java:1260)
        at org.jfree.chart.JFreeChart.createBufferedImage(JFreeChart.java:1453)
        at org.jfree.chart.JFreeChart.createBufferedImage(JFreeChart.java:1434)
        at org.jfree.chart.ChartUtils.writeChartAsPNG(ChartUtils.java:184)
        at org.jfree.chart.ChartUtils.saveChartAsPNG(ChartUtils.java:310)
        at org.jfree.chart.ChartUtils.saveChartAsPNG(ChartUtils.java:285)
        at com.github.frankivo.Graphs.createMonthGraph(Graphs.scala:53)
        at com.github.frankivo.Graphs$$anonfun$receive$1.applyOrElse(Graphs.scala:30)
        at akka.actor.Actor.aroundReceive(Actor.scala:537)
        at akka.actor.Actor.aroundReceive$(Actor.scala:535)
        at com.github.frankivo.Graphs.aroundReceive(Graphs.scala:26)
        at akka.actor.ActorCell.receiveMessage(ActorCell.scala:577)
        at akka.actor.ActorCell.invoke(ActorCell.scala:547)
        at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
        at akka.dispatch.Mailbox.run(Mailbox.scala:231)
        at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,5 +11,6 @@ COPY src src
 RUN sbt assembly
 
 FROM openjdk:8-alpine
-COPY --from=compiler /root/target/scala-2.13/*jar /root
+RUN apk add --no-cache ttf-dejavu
 CMD java -jar /root/*jar
+COPY --from=compiler /root/target/scala-2.13/*jar /root

